packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -460,6 +460,11 @@ class ScrollbarPainter extends ChangeNotifier implements CustomPainter {
         || fadeoutOpacityAnimation.value == 0.0)
       return;
 
+    // Skip painting if there's no viewportDimension.
+    if (!_lastMetrics!.hasViewportDimension) {
+      return;
+    }
+
```

Comment:
This can be added to the short circuit statement above as another || condition of returning.